wordpress-mobile/wpxmlrpc
# Upgrade for Xcode 12

This PR updates the `pod` for Xcode 12 and Apple Silicon and modernizes the CI by:
- Using the latest version of CocoaPods for XC12/AS compatibility
- Using the latest version of bundler to avoid recently security concerns
- Using `fastlane` for CI actions instead of the CircleCI-specific DSL
- Updating the `.ruby-version` to a newer release to align with the version available in CircleCI

**To Test:**
- The automated tests will pass (one can't get status reported because it's removed in favour of a more descriptive name)
- Branch from this PR and create a new tag (that doesn't already exist). Ensure that the publish succeeds, and note the new build message in `#build-and-ship`.
